addons.conf

```diff
@@ -48,21 +44,19 @@ server {
     }
 
     location @olympia {
-        # We are using uwsgi in our dev, stage and prod environments
-        # but not yet for our local development. So let's simply do a
-        # proxy_pass here to redirect to the django runserver.
-        proxy_pass http://web;
-        proxy_set_header X-FORWARDED-PROTOCOL "ssl";
-        proxy_set_header Host $http_host;
-        proxy_redirect off;
-        proxy_set_header X-Real-IP $remote_addr;
+        uwsgi_pass  web;
+        include     uwsgi_params;
+        uwsgi_read_timeout 60;
+        uwsgi_connect_timeout 60;
 
-        proxy_connect_timeout 30;
-        proxy_read_timeout 30;
+        uwsgi_param   Host                  $http_host;
+        uwsgi_param   X-Real-IP             $remote_addr;
+        uwsgi_param   X-Forwarded-For       $proxy_add_x_forwarded_for;
+        uwsgi_param   X-Forwarded-Protocol  ssl;
+        uwsgi_param   Vary                  User-Agent;
```

Comment:
Production passes more params on, like these,

```
uwsgi_param HTTPS on;

uwsgi_param  QUERY_STRING       $query_string;
uwsgi_param  REQUEST_METHOD     $request_method;
uwsgi_param  CONTENT_TYPE       $content_type;
uwsgi_param  CONTENT_LENGTH     $content_length;

uwsgi_param  REQUEST_URI        $request_uri;
uwsgi_param  PATH_INFO          $document_uri;
uwsgi_param  DOCUMENT_ROOT      $document_root;
uwsgi_param  SERVER_PROTOCOL    $server_protocol;

uwsgi_param  REMOTE_ADDR        $remote_addr;
uwsgi_param  REMOTE_PORT        $remote_port;
uwsgi_param  SERVER_PORT        $server_port;
uwsgi_param  SERVER_NAME        $server_name;
```

Not sure locally the app needs all those or not.

Reply:
Most of these params are passed on via `include uwsgi_params` and defined there (https://github.com/nginx/nginx/blob/master/conf/uwsgi_params). So I think this should be good.